common/lib/capa/capa/tests/response_xml_factory.py

```diff
@@ -267,9 +271,6 @@ def create_response_element(self, **kwargs):
 
         *answer_attr*: The "answer" attribute on the tag itself (treated as an
         alias to "expect", though "expect" takes priority if both are given)
-
-        *group_label*: Text to represent group of inputs when there are
```

Comment:
@cahrens The group label is like main question text and then individual "inputs" have their own label attribute. You can check detailed version here https://openedx.atlassian.net/wiki/display/A11Y/Custom+Python+Evaluated+Input 
Let me know if you still have any question(s).